Assets/Scripts/Utility/RDBLayout.cs

```diff
@@ -1211,7 +1211,7 @@ record = obj.Resources.FlatResource.TextureRecord;
                 // Setup standard billboard and assign RDB data
                 go = GameObjectHelper.CreateDaggerfallBillboardGameObject(archive, record, parent);
                 go.transform.position = targetPosition;
-                DaggerfallBillboard dfBillboard = go.GetComponent<DaggerfallBillboard>();
+                Billboard dfBillboard = go.GetComponent<Billboard>();
                 dfBillboard.SetRDBResourceData(obj.Resources.FlatResource);
```

Comment:
I think the `SetRDBResourceData` part should be moved out of the "if" (with an extra null check on the Billboard component of course), so mods have access to the RDB resource info.